benchmarks/helpers.py

```
@@ -0,0 +1,273 @@
+
+    Falls back to plain ``subprocess.run`` if sandbox-exec is unavailable.
+    """
+    if is_sandbox_exec_available():
+        # Write a temporary profile
+        with tempfile.NamedTemporaryFile(
+            mode="w", suffix=".sb", delete=False
+        ) as fp:
+            fp.write(_SEATBELT_PROFILE)
+            profile_path = fp.name
+        try:
+            return subprocess.run(
```

Ruff (0.15.4)

[error] `subprocess` call: check for execution of untrusted input

(S603)